services/app-api/src/emailer/templateHelpers.ts

```diff
@@ -0,0 +1,137 @@
+import * as Eta from 'eta'
+import * as path from 'path'
+
+import {
+    LockedHealthPlanFormDataType,
+    UnlockedHealthPlanFormDataType,
+    SubmissionType,
+} from '../../../app-web/src/common-code/healthPlanFormDataType'
+import { EmailConfiguration, StateAnalystsEmails } from '.'
+
+// ETA SETUP
+Eta.configure({
+    cache: true, // Make Eta cache templates
+    views: [
+        path.join(__dirname, 'etaTemplates'),
+        path.join(__dirname, '../../'), // this is intentional - we need to have this path here to be able to find the files from the graphql lambda when in serverless deployed app
+    ],
+})
+
+const renderTemplate = async <T>(
+    templateName: string,
+    data: T,
+    inUnitTest?: boolean
+) => {
+    if (!/^[a-zA-Z0-9]+$/.test(templateName)) {
+        console.error(
+            'CODING ERROR: templateName parameter should not include any punctuation, can only be alphanumeric characters'
+        )
+        return new Error(`${templateName} is not a valid template file name`)
+    }
+    const templatePath = `./${templateName}`
+
+    try {
+        const templateOrVoid = await Eta.renderFile(templatePath, data)
+
+        if (typeof templateOrVoid !== 'string') {
```

Comment:
This is fine but you could use a typescript type assertion function here https://mariusschulz.com/blog/assertion-functions-in-typescript